sdk/java-sdk/src/main/scala/com/akkaserverless/javasdk/impl/AnySupport.scala

```diff
@@ -390,7 +393,7 @@ class AnySupport(
 
       case other =>
         throw SerializationException(
-          s"Don't know how to serialize object of type ${other.getClass}. Try passing a protobuf, using a primitive type, or using a type annotated with @Jsonable.")
+          s"Don't know how to serialize object of type ${other.getClass}. Try passing a protobuf, using a primitive type, or defining a codec for this type.")
```

Comment:
And same here, no public API for defining "codec", maybe should just say `"Don't know how to serialize object of type ${other.getClass}."` without the recommendation instead?